src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
     this.maxRequests = 20;
 
     this.lastGeocode = new Date();
     this.currentRequests = 0;
 
     this.cache = new GeoCache(options.cacheFile);
-    this.geocoder = geocoder.init(options);
+    this.geocoder = geocoder.init({
+      clientId: options.clientId,
+      privateKey: options.privateKey,
+      key: options.apiKey
+    });
```

Comment:
`key: options.apiKey`. Unfortunately, in the `googlemaps` package the `apikey` option is not called `apiKey`, but just `key`. I found this so counter-intuitive and imprecise that I chose to change the name of the option from `key` to `apiKey` instead of passing it through directly. I can change this if you prefer the option to be just called `key`. In my opinion this is not a good name as it could easily be confused with the `privateKey` option.